Exercises/common.cmake

```diff
@@ -6,4 +6,18 @@ if(SPACK_CXX)
   set(ENV{CXX} ${SPACK_CXX})
 endif()
 
+include(FetchContent)
+if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.24)
+  # try find_package first before trying to download Kokkos
+  set(FETCH_CONTENT_EXTRA_ARGS FIND_PACKAGE_ARGS NAMES Kokkos)
```

Comment:
Nice, I didn't know about this one.
As always, reading CMake documentation is a pain. [Relevant info](https://cmake.org/cmake/help/latest/module/FetchContent.html#variable:FETCHCONTENT_TRY_FIND_PACKAGE_MODE) for anyone interested.